internal/access/identity.go

```diff
@@ -34,6 +34,23 @@ func GetIdentity(c *gin.Context, id uid.ID) (*models.Identity, error) {
 	return data.GetIdentity(rCtx.DBTxn, data.GetIdentityOptions{ByID: id, LoadProviders: true})
 }
 
+func GetIdentityByName(c *gin.Context, name string) (*models.Identity, error) {
```

Comment:
Suggestion: I wonder if we could move the `GetIdentityOptions` to the caller in `server/users.go` and use one `access.GetIdentity` function that take the `GetIdentityOptions` as an argument.

Reply:
I like this suggestion. We've done this in a few other access functions already and it works well.